docs/tasks/issuers/setup-acme/dns01/azuredns.rst

```diff
@@ -53,8 +53,11 @@ You can configure the issuer like so:
        - dns01:
            azuredns:
              # Service principal clientId (also called appId)
+             # Optional. If omitted, a Managed Identity will be used for Azure API authentication.
+             # Requires aadPodIdentity to be configured in the cluster
              clientID: AZURE_SERVICE_PRINCIPAL_ID
              # A secretKeyRef to a service principal ClientSecret (password)
+             # Required if clientID is set
```

Comment:
We have moved our documentation to github.com/cert-manager/website - would you mind opening a PR against [this](https://github.com/cert-manager/website/blob/master/content/en/docs/configuration/acme/dns01/azuredns.md) file with theese changes? 😄

Reply:
It'd also be great if you could add a header/section specifically relating to using managed identity, i.e. `Authenticating using Azure Managed Identity` (with a brief blurb explaining why you'd do this, e.g. `In order to avoid managing a client ID and client secret for Azure, it is possible to authenticate to Azure using 'ambient credentials' which use [Azure Managed Identity](link here) to authenticate with the Azure API ...`)